migrations/helpers/dart_samples_update_helper.py

```diff
@@ -150,16 +151,13 @@ def migrate_all_dbs(config, s_start_datetime: str = "", s_end_datetime: str = ""
         logger.exception(e)
 
 
-def get_positive_samples(
-    samples_collection: Collection, start_datetime: datetime, end_datetime: datetime
-) -> List[Sample]:
+def get_samples(samples_collection: Collection, start_datetime: datetime, end_datetime: datetime) -> List[Sample]:
     logger.debug(f"Selecting positive samples between {start_datetime} and {end_datetime}")
```

Comment:
Think this debug message should have the word 'positive' removed

Reply:
True.

Reply:
Will address this in a new, small PR